packages/react/src/components/label/label.react-template.tsx

```diff
@@ -8,15 +8,18 @@ export const reactLabel: ComponentImplementation<Label> = {
   component: "label",
   implementation: "react",
   template: () =>
-    function labelTemplate({ status, label, removable, dsoRemoveClick, compact, truncate, symbol }) {
+    function labelTemplate({ slotName, status, label, removable, dsoRemoveClick, compact, truncate, symbol }) {
+      const props = {
+        ...(slotName && { slot: slotName }),
+        status,
+        onDsoRemoveClick: dsoRemoveClick,
+        compact,
+        truncate,
+        removable,
+      };
+
       return (
-        <DsoLabel
-          status={status}
-          onDsoRemoveClick={dsoRemoveClick}
-          compact={compact}
-          truncate={truncate}
-          removable={removable}
-        >
+        <DsoLabel {...props}>
```

Comment:
Ik denk dat ik voorkeur heb aan de oude opzet, omdat dan wat duidelijke is of props ook echt missen/mogen.

Daarnaast is dit een beetje dubbelop. In `labelTemplate()` destructuren we `props` om vervolgens weer `props` te maken.

Reply:
Dan doen we alleen de slot prop op deze manier.